resources/icons/16px/app-pairing.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M2 3V9C2.55228 9 3 9.44772 3 10C3 10.5523 2.55228 11 2 11H1.25C0.559644 11 0 10.4404 0 9.75L0 2.25C0 1.55965 0.559643 1 1.25 1H10.75C11.4404 1 12 1.55965 12 2.25V9.75C12 10.4404 11.4404 11 10.75 11H8C7.44772 11 7 10.5523 7 10C7 9.44772 7.44772 9 8 9H10V3H2Z" fill="#111418"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M14 13V7C13.4477 7 13 6.55228 13 6C13 5.44771 13.4477 5 14 5H14.75C15.4404 5 16 5.55964 16 6.25V13.75C16 14.4404 15.4404 15 14.75 15H5.25C4.55964 15 4 14.4404 4 13.75V6.25C4 5.55965 4.55964 5 5.25 5H8C8.55228 5 9 5.44771 9 6C9 6.55228 8.55228 7 8 7H6V13H14Z" fill="#111418"/>
+</svg>
```

Comment:
Can we remove the `width`, `height`, `fill`, `fill-rule`, and `clip-rule`, properties from svg and path? They aren't necessary in this instance:
```svg
<svg viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
<path d="M2 3V9C2.55228 9 3 9.44772 3 10C3 10.5523 2.55228 11 2 11H1.25C0.559644 11 0 10.4404 0 9.75L0 2.25C0 1.55965 0.559643 1 1.25 1H10.75C11.4404 1 12 1.55965 12 2.25V9.75C12 10.4404 11.4404 11 10.75 11H8C7.44772 11 7 10.5523 7 10C7 9.44772 7.44772 9 8 9H10V3H2Z"/>
<path d="M14 13V7C13.4477 7 13 6.55228 13 6C13 5.44771 13.4477 5 14 5H14.75C15.4404 5 16 5.55964 16 6.25V13.75C16 14.4404 15.4404 15 14.75 15H5.25C4.55964 15 4 14.4404 4 13.75V6.25C4 5.55965 4.55964 5 5.25 5H8C8.55228 5 9 5.44771 9 6C9 6.55228 8.55228 7 8 7H6V13H14Z"/>
</svg>
```
<img src="https://github.com/user-attachments/assets/0cef5d62-9f07-4309-92ac-eb7cfca71adc" alt="app-pairing" width="16" />

Reply:
done!